Services/DataBaseService/Database/Migrations/Scripts/04.2020/0004_CreateUsersAvatarsTable.sql

```diff
@@ -1,7 +1,7 @@
 ﻿CREATE TABLE dbo.UsersAvatars (
 	[Id] UNIQUEIDENTIFIER UNIQUE NOT NULL,
-	[Avatar] VARBINARY(MAX) NOT NULL,
```

Comment:
Why it can be null?

Reply:
Because User can change avatar to default one